pkgs/development/web/rover/etag.id

```diff
@@ -0,0 +1 @@
+a7252784c3d195f1ffec315fb8b1481cfba06c7943fe9020ab8df2aafd4d479e
```

Comment:
Why do we need to write that into a file? Why not just use a var?

Reply:
Good point, changed it to a var in 41801e346d0175372c906f75c782eb20e706199f